nexus/src/db/datastore.rs

```diff
@@ -2679,6 +2679,70 @@ impl DataStore {
             }),
         }
     }
+
+    // SSH public keys
+
+    pub async fn ssh_keys_list(
+        &self,
+        silo_user_id: Uuid,
+        page_params: &DataPageParams<'_, Name>,
+    ) -> ListResultVec<SshKey> {
+        use db::schema::ssh_key::dsl;
+
```

Comment:
I think it's be more consistent with the existing methods if:

- the caller was responsible for doing a lookup of silo_user_id first and passed that in as `authz_user: authz::SiloUser` rather than a uuid
- we did an authz check here using `opctx.authorize(authz::Action::ListChildren, authz_user).await?)`

Reply:
Fixed in 95114e05, thanks. LMK if you object to the change to `Nexus::silo_user_fetch` here.